app/controllers/giveaways_controller.rb

```diff
@@ -1,6 +1,9 @@
 class GiveawaysController < ApplicationController
+  def home
+  end
+
   def index
-    @giveaways = Giveaway.all
+    @giveaways = Giveaway.all.reverse
   end
```

Comment:
Can't we use the index page as a homepage?

Reply:
@violeta-p Yes we can. Thanks for the suggestion.